command.c

```diff
@@ -1,7 +1,7 @@
 #include <ctype.h>
 #include <errno.h>
 #include <hiredis/alloc.h>
-#include <string.h>
+#include <strings.h>
```

Comment:
How could this have worked with the wring header? They don't define the same functions, e.g. string.h is for strcmp() while strings.s is for strcasecmp()...

Reply:
The strangest thing, its only triggered when building with makefiles aswell.
I see that CMake automatically sets `gnu99` when using the 99 standard,
while the makefile uses `c99` (as in hiredis and hiredis-vip) which also requires `_XOPEN_SOURCE=600`. 
This maybe trigger different indirect includes

Reply:
As long as it works in all kinds of builds, I trust you.